source/frontend/src/features/mapSideBar/acquisition/tabs/compensation/update/models.ts

```diff
@@ -79,6 +81,13 @@ export class CompensationRequisitionFormModel {
     compensation.serviceLine = apiModel.chartOfAccountsId?.toString() || '';
     compensation.chartOfAccounts = apiModel.chartOfAccounts;
     compensation.responsibilityCentre = apiModel.responsibilityId?.toString() || '';
+    compensation.alternateProject =
+      apiModel.alternateProject !== null
+        ? {
+            id: apiModel.alternateProject?.id || 0,
```

Comment:
this should probably be changed, if the alternate project id is not set we should not be setting the alternate project.